### PR TITLE
Add rpc interface for arbiter

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: servers/interfaces.go [hunk 1]
```diff
@@ -829,3 +984,53 @@ func ResponsePack(errCode ErrCode, result interface{}) map[string]interface{} {
 	}
 	return map[string]interface{}{"Result": result, "Error": errCode}
```
Comment (AlexpanXX) [inline servers/interfaces.go]: use common method ResponsePack() method

File: servers/interfaces.go [hunk 2]
```diff
@@ -92,6 +94,129 @@ func GetTransactionInfo(header *Header, tx *ela.Transaction) *TransactionInfo {
...
+	case TransferCrossChainAssetInfo:
+		obj := new(ela.PayloadTransferCrossChainAsset)
+		obj.AddressesMap = object.AddressesMap
+		return obj, nil
+	}
+
+	return nil, errors.New("Invalid payload type.")
+}
+
+func GetTransaction(txinfo *TransactionInfo) (*ela.Transaction, error) {
```
Comment (AlexpanXX) [inline servers/interfaces.go]: use txInfo instead of txinfo

File: servers/interfaces.go (hunk 1 above)
Comment (RainFallsSilent) [reply servers/interfaces.go]: GetTransaction() and GetBlockTransactionsDetail() function is used not for rpc interface, no need to use common method ResponsePack() method

Comment (RainFallsSilent) [reply servers/interfaces.go]: txinfo has changed to txInfo

Comment (AlexpanXX) [reply servers/interfaces.go]: GetBlockByHeight() is the method need to use common ResponsePack() method

**Pull request merged**